package.json

```diff
@@ -81,58 +81,56 @@
     "build:companion": "cd ./packages/@uppy/companion && npm run build",
     "build:css": "node ./bin/build-css.js",
-    "build:gzip": "node ./bin/gzip.js",
     "build:js": "npm-run-all build:lib build:bundle",
     "build:lib": "node ./bin/build-lib.js",
-    "build": "npm-run-all --parallel build:js build:css build:companion --serial build:gzip size",
-    "clean": "rm -rf packages/*/lib packages/@uppy/*/lib packages/*/dist packages/@uppy/*/dist",
-    "contributors:save": "replace-x -m '<!--contributors-->[\\s\\S]+<!--/contributors-->' \"<!--contributors-->\n## Contributors\n\n$(npm run --silent contributors)\n<!--/contributors-->\" README.md",
-    "contributors": "githubcontrib --owner transloadit --repo uppy --cols 6 $([ \"${GITHUB_TOKEN:-}\" == \"\" ] && echo \"\" || echo \"--authToken ${GITHUB_TOKEN}\") --showlogin true --sortOrder desc",
+    "build": "npm-run-all --parallel build:js build:css build:companion --serial size",
+    "contributors:fetch": "githubcontrib --owner transloadit --repo uppy --cols 6 $([ \"${GITHUB_TOKEN:-}\" == \"\" ] && echo \"\" || echo \"--authToken ${GITHUB_TOKEN}\") --showlogin true --sortOrder desc",
+    "contributors:save": "replace-x -m '<!--contributors-->[\\s\\S]+<!--/contributors-->' \"<!--contributors-->\n## Contributors\n\n$(npm run --silent contributors:fetch)\n<!--/contributors-->\" README.md",
     "dev:browsersync": "browser-sync start --no-open --no-ghost-mode false --server examples/dev --port 3452 --serveStatic packages/uppy/dist --files \"examples/dev/bundle.js, packages/uppy/dist/uppy.min.css, packages/uppy/lib/**/*\"",
-    "dev:js": "cd examples/dev && npm run watch",
-    "dev:no-companion": "npm-run-all --parallel dev:js watch:css watch:js:lib dev:browsersync",
-    "dev": "npm-run-all --parallel start:companion dev:js watch:css watch:js:lib dev:browsersync",
+    "dev:watch-sandbox": "cd examples/dev && npm run watch:sandbox",
+    "dev:with-companion": "npm-run-all --parallel start:companion dev:watch-sandbox watch dev:browsersync",
+    "dev": "npm-run-all --parallel dev:watch-sandbox watch dev:browsersync",
     "lint:fix": "npm run lint -- --fix",
     "lint:staged": "lint-staged",
     "lint": "eslint . --cache",
     "release": "./bin/release",
     "size": "echo 'JS Bundle mingz:' && cat ./packages/uppy/dist/uppy.min.js | gzip | wc -c && echo 'CSS Bundle mingz:' && cat ./packages/uppy/dist/uppy.min.css | gzip | wc -c",
-    "start:companion": "sh ./bin/companion",
-    "start": "npm-run-all --parallel watch start:companion web:preview",
-    "test:acceptance:local": "npm run test:build && wdio test/endtoend/wdio.local.conf.js",
-    "test:acceptance": "npm run test:prepare-ci && wdio test/endtoend/wdio.remote.conf.js",
-    "test:build-ci": "./bin/endtoend-build-ci",
-    "test:build": "./bin/endtoend-build",
+    "start:companion": "./bin/companion",
+    "start": "npm-run-all --parallel watch start:companion web:start",
```

Comment:
What if I want to work on the website without companion? AJ working on a post doesn’t need Companion, probably. `web:start` for that?

Reply:
`web:start` I’d say yes